### ISSUE TITLE
1.8.8 PaperSpigot 

### ISSUE THREAD

**Issue by Neeonn** [YouHaveTrouble/CommandWhitelist#18]
**Describe the bug**
Plugin doesn't work as intended. 

**Expected behavior**
Block commands & subcommands.

**To Reproduce**
Install PaperSpigot 1.8.8 (443), install ProtocolLib 4.6.0, install CommandWhitelist 1.7.1, try my config.

**Screenshots**
![image](https://user-images.githubusercontent.com/24417584/110943721-d4b6f000-833b-11eb-9453-55734767b9b0.png)
![image](https://user-images.githubusercontent.com/24417584/110943768-e3050c00-833b-11eb-98f6-8a667ec42280.png)

**Additional context**
```
[14:05:48 INFO]: [CommandWhitelist] Enabling CommandWhitelist v1.7.1
[14:05:48 ERROR]: [CommandWhitelist] Plugin CommandWhitelist v1.7.1 has failed to register events for class eu.endermite.commandwhitelist.spigot.listeners.PlayerCommandSendListener because org/bukkit/event/player/PlayerCommandSendEvent does not exist.
[14:05:48 INFO]: [CommandWhitelist] §bRunning in legacy mode...
[14:05:48 ERROR]: [CommandWhitelist] Plugin CommandWhitelist v1.7.1 has failed to register events for class eu.endermite.commandwhitelist.spigot.listeners.TabCompleteBlockerListener because org/bukkit/event/server/TabCompleteEvent does not exist.
```

**Comment by YouHaveTrouble** [comment]
Seems like adding protocollib mode for modern version broke legacy mode check. I will fix that, but it's not my priority to support legacy, so it might take some time before I get to it.

**Comment by Neeonn** [comment]
Alright, thank you for the answer. Appreciate the work.

**Comment by YouHaveTrouble** [comment]
Fixed in [1.7.2](https://github.com/YouHaveTrouble/CommandWhitelist/releases/tag/v1.7.2).
